relay/metadata_provider.go

```diff
@@ -153,8 +155,15 @@ func (m *metadataProvider) fetchMetadata(key v2.BlobKey) (*blobMetadata, error)
 		}
 	}
 
+	blobSize := uint32(cert.BlobHeader.BlobCommitments.Length)
```

Comment:
`cert.BlobHeader.BlobCommitments.Length` gives you the blob length in symbols. 
I think what we want is here `cert.BlobHeader.BlobSize` (size of raw blob data in bytes) instead

Reply:
There is currently no such field. Is that something that is going to be added?